### PR TITLE
added sanity check for cert experation

### PR DESCRIPTION
**Pull request by eran-braun**

### CODE REVIEW

File: janus.c [hunk 1]
```diff
@@ -3551,7 +3552,50 @@ gboolean carbyne_janus_transport_is_sanityhealthcheck_resources_available(janus_
 		JANUS_LOG(LOG_WARN,"SGW not Available\n");
 	}
 
-  	return TRUE;
+
+	//checking if cert is ok
+	FILE *certStatusFile = NULL;
+	char * certStatusLine = NULL;
```
Comment (alex-dizengof) [inline janus.c]: fix coding style

Comment (alex-dizengof) [reply janus.c]: char *certStatusLine

File: janus.c [hunk 2]
```diff
@@ -3551,7 +3552,50 @@ gboolean carbyne_janus_transport_is_sanityhealthcheck_resources_available(janus_
...
+	if (certStatusFile == NULL)
+	{
+		JANUS_LOG(LOG_ERR,"Failed opening cert status file\n");
+		certStatusStatus = FALSE;
+	}
+	else
+	{
+		if((getline(&certStatusLine, &certStatuslen, certStatusFile)) != -1) 
+		{
+			if(strstr(certStatusLine,"true") == NULL)
```
Comment (alex-dizengof) [inline janus.c]: identation

File: janus.c [hunk 3]
```diff
@@ -3551,7 +3552,50 @@ gboolean carbyne_janus_transport_is_sanityhealthcheck_resources_available(janus_
...
+			JANUS_LOG(LOG_ERR,"no line was found in cert status file\n");
+			certStatusStatus = FALSE;
+		}
+	}
+	// free memeory
+	if (certStatusFile != NULL) 
+	{
+		fclose(certStatusFile);
+	}
+	if (certStatusLine != NULL)
```
Comment (alex-dizengof) [inline janus.c]: extra space

File: janus.c [hunk 4]
```diff
@@ -3551,7 +3552,50 @@ gboolean carbyne_janus_transport_is_sanityhealthcheck_resources_available(janus_
...
+	{ 
+		free(certStatusLine);
+	}
+
+	// check status at the end
+	if (certStatusStatus != TRUE)
```
Comment (alex-dizengof) [inline janus.c]: you can just return certStatusStatus

Comment (eran-braun) [reply janus.c]: instead of returning TRUE as well?
if so i will need to also address all other tests in the function

**Review by assafbc — APPROVED** [review]

**Pull request merged**